client/src/components/Navbar/Menus/Unauthenticated/index.tsx

```diff
@@ -0,0 +1,35 @@
+import React from 'react';
+import { Button, Grid, makeStyles } from '@material-ui/core';
+import { useHistory } from 'react-router';
+
+const useStyles = makeStyles({
+  button: {
+    color: 'white',
+    fontWeight: 'bolder',
+    border: '1px solid white',
+    padding: '15px 40px',
+    borderRadius: 0,
+    letterSpacing: 1,
+  },
+});
+
+export const index = (): JSX.Element => {
+  const classes = useStyles();
+  const history = useHistory();
+  const [location, setLocation] = React.useState(history.location.pathname);
+
+  return (
+    <Grid item>
+      <Button
+        className={classes.button}
+        onClick={() => {
+          const newLocation = location === '/login' ? '/signup' : '/login';
```

Comment:
instead of leading them to a new page with js, because it wouldn't be good for accessibility, you may want to look into react router for generating a button that would link you to the /login or /signup page